src/stream/src/executor/now.rs

```diff
@@ -406,19 +386,33 @@ mod tests {
     ) -> (UnboundedSender<Barrier>, BoxedMessageStream) {
         let table_id = TableId::new(1);
         let column_descs = vec![ColumnDesc::unnamed(ColumnId::new(0), DataType::Timestamptz)];
-        let order_types = create_order_types();
-        let pk_indices = create_pk_indices();
         let state_table = StateTable::new_without_distribution(
             state_store.clone(),
             table_id,
             column_descs,
-            order_types,
-            pk_indices,
+            vec![],
+            vec![],
         )
         .await;
 
         let (sender, barrier_receiver) = unbounded_channel();
-        let now_executor = NowExecutor::new(barrier_receiver, 1, state_table);
+
+        let schema = Schema::new(vec![Field {
+            data_type: DataType::Timestamptz,
+            name: String::from("now"),
+            sub_fields: vec![],
+            type_name: String::default(),
+        }]);
+
+        let now_executor = NowExecutor::new(
+            ExecutorInfo {
+                schema,
+                pk_indices: vec![],
+                identity: "NowExecutor".to_string(),
+            },
```

Comment:
I'm considering whether we should encapsulate the constructors into methods, in case that newly introduced fields in the future (like `op_info` #5940) will lead to significant changes in tests. But it's okay to do this refactoring in the future.